bindings/pyroot_experimental/PyROOT/test/root_module.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+
+class ROOTModule(unittest.TestCase):
+    def test_import(self):
```

Comment:
Can we have a docstring also at class level explaining what this class of tests does? Which is basically testing the functionality provided by the ROOT module facade.
E.g.
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/PyROOT/test/rdataframe_makenumpy.py#L8

Reply:
And done!